app/src/main/java/ai/elimu/appstore/synchronization/AppListAdapter.java

```diff
@@ -464,6 +470,16 @@ public void onUninstallComplete(@NonNull String packageName) {
                     }
                 }
             });
+
+            //Uninstall an application
+            holder.btnUninstall.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    Intent uninstallIntent = new Intent(Intent.ACTION_UNINSTALL_PACKAGE);
```

Comment:
Have you tested that this solution also works with Android P?: https://developer.android.com/reference/android/content/Intent.html#ACTION_UNINSTALL_PACKAGE

Reply:
The permission `REQUEST_DELETE_PACKAGES` mentioned in above pasted link is only required if our app target API P or up. However, our current `targetSdkVersion` is 26. That means we don't need the permission to be implemented.

I have also tested with Android P. Uninstall function just works fine.

Reply:
Ok. And what will happen the day we upgrade the app to use `targetSdkVersion` 28?

Reply:
Yes, when targeting APIs P or greater, we'll need the permission `REQUEST_DELETE_PACKAGES`.
I just added the permission to Manifest file so that Uninstall still works when we decide to target API P. 
Tested in P and works fine.